data/core/encyclopedia/geography.cfg

```diff
@@ -233,12 +236,19 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
 [topic]
     id=far_south
     title= _ "Far South"
-    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is also inhabited by many local creatures, including the Jinn and Wyverns." + "
-
+    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is also inhabited by many local creatures, including the Jinn and Wyverns. Very little is known for certain about this land." + "
```

Comment:
I just realized that Kerlath is mentioned several times, yet never clearly defined. (It looks like it's implicitly defined in Southwest Elven Lands?)

Reply:
Kerlath is mentioned as a province in Southern Wesnoth in the Westin description, though that is quite vague. I might replace it with the more recognizable Wesnoth. I could also describe it more explicititly in the Wesnoth description as well. (btw thanks for fixing all the indenation and breaking up the strings)